sys/vane/rver.hoon

```diff
@@ -60,7 +60,7 @@
       $:  %b
           ::
           ::
-          $%  [%wake ~]
+          $%  [%wake error=(unit tang)]
```

Comment:
`+take` needs to actually handle this error case. See https://github.com/urbit/arvo/pull/1192#discussion_r293957948